docs/node-operators/migration-guides/upgrade-mainnet-to-0714.md

```diff
@@ -0,0 +1,663 @@
+2. `socketPath` - path to the Unix sock file that Vega creates for communicating with other network components running locally. 
+
+Make sure the parent directory for the `sock` file exists. To check that it does, run: `mkdir -p <USER-HOME>/run/vega.sock`
+
+
+### 14. Prepare systemd service for Visor
+Create the `/lib/systemd/system/vegavisor.service` file with the following content. The file **must** be created with the `root` or a higher permission user:
+
+```toml
+# /lib/systemd/system/vegavisor.service
+
+[Unit]
+Description=vegavisor
+Documentation=https://github.com/vegaprotocol/vega
+After=network.target network-online.target
+Requires=network-online.target
+
+[Service]
+User=<VEGA-USER>
+Group=<VEGA-USER>
+ExecStart="<VISOR-BIN>" run --home "<VEGAVISOR-HOME>"
+TimeoutStopSec=10s
+LimitNOFILE=1048576
+LimitNPROC=512
+PrivateTmp=true
+ProtectSystem=full
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+
+[Install]
+WantedBy=multi-user.target
+```
+
+
+### 15. Create Vega and data node systemd services
+If you are not using Visor, you have to prepare similar systemd configs as seen in the step above, for the following services:
+- Vega (Tendermint is part of the Vega process in newer versions)
+- Data node (optional)
+
+1. Create the vega systemd service under the location: `/lib/systemd/system/vega.service`
+2. Create the data-node systemd file (if you run a data node): `/lib/systemd/system/data-node.service`
+
+#### Example of the Vega service file
+
+```toml
+# /lib/systemd/system/vega.service
+
+[Unit]
+Description=vega
+Documentation=https://github.com/vegaprotocol/vega
+After=network.target network-online.target
+Requires=network-online.target
+
+[Service]
+User=vega
+Group=vega
+ExecStart="<VEGA-BIN>" start --home "<VEGA-NETWORK-HOME>" --tendermint-home "<TENDERMINT-HOME>" --nodewallet-passphrase-file "<VEGA-NETWORK-HOME>/all-wallet-passphrase.txt"
+TimeoutStopSec=10s
+LimitNOFILE=1048576
+LimitNPROC=512
+PrivateTmp=true
+ProtectSystem=full
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+
+[Install]
+WantedBy=multi-user.target
+```
+
+#### Example of the data node service file
+
+```
+# /lib/systemd/system/data-node.service
+
+[Unit]
+Description=tendermint
+Documentation=https://github.com/vegaprotocol/vega
+After=network.target network-online.target
+Requires=network-online.target
+
+[Service]
+User=vega
+Group=vega
+ExecStart="<VEGA-BIN>" datanode start --home "<VEGA-NETWORK-HOME>"
+TimeoutStopSec=10s
+LimitNOFILE=1048576
+LimitNPROC=512
+PrivateTmp=true
+ProtectSystem=full
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+
+[Install]
+WantedBy=multi-user.target
+```
+
+
+
+### 16. Install/Upgrade PostgreSQL for data node
+If you are running a data node, you will need to install or upgrade PostgreSQL. The PostgreSQL instance must be on the same server as the data node, as the PostgreSQL does data-snapshot in the data node location file space. The data node uses PostgreSQL snapshots for the network history feature.
```

Comment:
`The PostgreSQL instance must be on the same server as the data node, as the PostgreSQL does data-snapshot in the data node location file space`  -  The data snapshot process no longer relies on being in the same location file space, although having said that, afaik we have not tested thoroughly - so we should probably stick with the recommendation for now - but not for the reason given.